### ISSUE TITLE
Upgrade to V5.0 > undefined method `call' for nil:NilClas

### ISSUE THREAD

**Issue by lienvdsteen** [airbrake/airbrake#497]
* Airbrake version: 5.0.4
* Ruby version: 2.0.0
* Framework name & version: Rails 4.0.2

#### Airbrake config

```ruby
Airbrake.configure do |c|
  c.project_id = ***********
  c.project_key = ****************
  c.root_directory = Rails.root
  c.logger = Rails.logger
  c.environment = Rails.env
  c.ignore_environments = %w(test, development)
end
```

#### Description

We seem to be getting a lot of these `undefined method 'call' for nil:NilClass` errors. I checked with the other issues here and we upgraded passenger to the latests version. But this didn't fix the problem.

These are the _important_ lines of our Gemfile.lock:

```ruby
airbrake (5.0.4)
  airbrake-ruby (~> 1.0)
airbrake-ruby (1.0.4)
passenger (5.0.24)
      rack
      rake (>= 0.8.1)
```

If you need anything else or more information, I will provide it..

**Comment by kyrylo** [comment]
Hi! Please read https://github.com/airbrake/airbrake/issues/490#issuecomment-173926281

It's a Passenger bug and we simply report it (nothing to do with out library, we simply do our job).

**Comment by kyrylo** [comment]
By the way, your config is a bit wrong. This:

```ruby
c.ignore_environments = %w(test, development)
```

should be written like this:

```ruby
c.ignore_environments = %w(test development)
```

The comma is not needed if you use `%w()`.

**Comment by lienvdsteen** [comment]
Thank you! And sorry for the duplicate bug

**Comment by kyrylo** [comment]
You are welcome, let me know if you have troubles. The new passenger was released two days ago (5.0.24), so I assume the fix is there.
